tutorials/nlp/Token_Classification-BioMegatron.ipynb

```diff
@@ -569,9 +569,6 @@
     "    text_file=os.path.join(DATA_DIR, 'NER', 'sample_text_dev.txt'),\n",
     "    labels_file=os.path.join(DATA_DIR, 'NER', 'sample_labels_dev.txt'),\n",
     "    output_dir=exp_dir,\n",
-    "    add_confusion_matrix=True,\n",
```

Comment:
Is it OK to drop these? The user do not need to see the confusion matrix?

Reply:
apparently that can cause some import error on some setting (not sure what it is), and that's how the NER Notebook does too: https://github.com/NVIDIA/NeMo/blob/main/tutorials/nlp/Token_Classification_Named_Entity_Recognition.ipynb

Reply:
When it is enabled, does it show the confusion matrix properly?
How about bringing them back and add a comment on the top of the tutorial on the version requirement of ipython?

Reply:
Yes, it works fine on Colab, for example.
Excellent suggestion :) I've put it back and added a comment pointing to StackOverflow for the error about the Matplotlib version incompatibility.